tests/test_cli.py

```diff
@@ -12,9 +11,9 @@
 
 @pytest.mark.parametrize('args', [
     ['composer', '--version'],
-    [sys.executable, '-m', 'composer', '--version'],
-    [sys.executable, '-m', 'composer.cli', '--version'],
-    [sys.executable, '-m', 'composer.cli.launcher', '--version'],
+    ['python', '-m', 'composer', '--version'],
```

Comment:
unintentional?

Reply:
clean up from chuck -- python is better than sys executable imo